Tests/UnitTests/Purchasing/ErrorUtilsTests.swift

```diff
@@ -38,6 +38,13 @@ class ErrorUtilsTests: TestCase {
         super.tearDown()
     }
 
+    func testErrorsCanBeConvertedToErrorCode() throws {
+        let error = ErrorUtils.customerInfoError()
```

Comment:
I think this should be enough, but should we do this for all error types? It might be a lot of tests though 😅

Reply:
Yeah I think that's not necessary. This is indirectly testing `ErrorUtils.error(with:message:...)`, but that's `private`.